util/src/main/java/io/kubernetes/client/Configmap.java

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client;
+
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.Configuration;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.V1ConfigMap;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.util.exception.CreateConfigmapException;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import okhttp3.Call;
+import okhttp3.Response;
+
+/** Utility class to create Configmaps */
+public class Configmap {
+
+  private ApiClient apiClient;
+  private CoreV1Api coreClient;
+
+  public Configmap() {
+    this(Configuration.getDefaultApiClient());
+  }
+
+  public Configmap(ApiClient apiClient) {
+    this.apiClient = apiClient;
+    this.coreClient = new CoreV1Api(apiClient);
+  }
+
+  public ApiClient getApiClient() {
+    return apiClient;
+  }
+
+  public void setApiClient(ApiClient apiClient) {
+    this.apiClient = apiClient;
+  }
+
+  /**
+   * Method to create a Configmap from property file. Target file must be in .properties format and
+   * should contain data in key=value format This is equivalent to kubectl create configmap
+   * {configmap_name} --from-file={file_path}
+   *
+   * @param configmapName
+   * @param filePath
+   * @return
+   * @throws IOException
+   * @throws ApiException
+   */
+  public String createConfigmapFromFile(String configmapName, Path filePath)
+      throws IOException, ApiException, CreateConfigmapException {
+    return createConfigmapFromFile("default", configmapName, filePath);
+  }
+
+  public String createConfigmapFromFile(String namespace, String configmapName, Path filePath)
```

Comment:
I think that we should either make this capable of loading any file, or rename this method `createConfigMapFromPropertiesFile(...)`

My preference would be to make this generic so that it works with any type of file.